.OpenAPIdocumentation.yaml

```diff
@@ -0,0 +1,107 @@
+openapi: 3.0.0
+info:
+  title: Ekadashi-server
+  version: 0.0.1
+paths:
+  /register:
+    post:
+      summary: Register a new user in ekadashi database.
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/RegisterRequest'
+      responses:
+        '200':
+          description: Registration succeed 
+        '400':
+          description: Incorrect username or password
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        '401': 
+          description: Incorrect username or password
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        '500': 
+          description: Internal Server Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+  /login:
+    post:
+      summary: Login user
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/LoginRequest'
+      responses:
+        '200':  
+          description: Login succeed 
```

Comment:
Need to describe `Set-Cookie`here, see https://swagger.io/docs/specification/authentication/cookie-authentication/